x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/import_export/export_prebuilt_rules.ts

```diff
@@ -0,0 +1,502 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import {
+  BulkActionTypeEnum,
+  RuleResponse,
+} from '@kbn/security-solution-plugin/common/api/detection_engine';
+import { FtrProviderContext } from '../../../../../../ftr_provider_context';
+import { deleteAllRules } from '../../../../../../../common/utils/security_solution';
+import {
+  binaryToString,
+  createPrebuiltRuleAssetSavedObjects,
+  createRuleAssetSavedObject,
+  deleteAllPrebuiltRuleAssets,
+  getCustomQueryRuleParams,
+  installPrebuiltRules,
+  parseNdJson,
+} from '../../../../utils';
+
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const securitySolutionApi = getService('securitySolutionApi');
+  const log = getService('log');
+
+  const PREBUILT_RULE_ID_A = 'test-prebuilt-rule-a';
+  const PREBUILT_RULE_ID_B = 'test-prebuilt-rule-b';
+  const [PREBUILT_RULE_A, PREBUILT_RULE_B] = [
+    createRuleAssetSavedObject({
+      rule_id: PREBUILT_RULE_ID_A,
+      tags: ['test-tag'],
+    }),
+    createRuleAssetSavedObject({
+      rule_id: PREBUILT_RULE_ID_B,
+      tags: ['test-tag-2'],
+    }),
+  ];
+  const PREBUILT_RULE_ASSETS = [PREBUILT_RULE_A, PREBUILT_RULE_B];
+
+  /**
+   * This test suite is skipped in Serverless MKI environments due to reliance on the
+   * feature flag for prebuilt rule customization.
+   */
+  describe('@ess @serverless @skipInServerlessMKI Export prebuilt rules', () => {
+    beforeEach(async () => {
+      await deleteAllPrebuiltRuleAssets(es, log);
+      await deleteAllRules(supertest, log);
+      await createPrebuiltRuleAssetSavedObjects(es, PREBUILT_RULE_ASSETS);
+      await installPrebuiltRules(es, supertest);
```

Comment:
Let's move the asset creation and installation that's important for test scenarios into the test body (inside each `it`). Having part of the state initialized in `beforeEach` and the rest in the test body makes it very hard to read and understand; I constantly need to scroll up and down 500 LoC just to figure out the initial state.

It's fine to do cleanup in the `beforeEach` section, but initializing important state there is an anti-pattern in my opinion.